src/middlewared/middlewared/etc_files/local/openvpn/server/perms.py

```diff
@@ -0,0 +1,29 @@
+import errno
+import os
+
+from pathlib import Path
+
+from middlewared.service import CallError
+
+
+LOGS_FOLDER = '/var/log/openvpn'
+
+
+def fix_perms(middleware):
+    if nobody_user := middleware.call_sync('user.query', [('username', '=', 'nobody')]):
+        nobody_uid = nobody_user[0]['uid']
+        nobody_gid = nobody_user[0]['group']['bsdgrp_gid']
+    else:
+        raise CallError('Unable to locate "nobody" user', errno=errno.ENOENT)
+
+    log_dir = Path(LOGS_FOLDER)
```

Comment:
There is no reason to define a global LOGS_FOLDER now. Just move it inside the function.